apps/i18n/studio/en_us.json

```diff
@@ -149,6 +149,7 @@
   "endGameWin": "end game as a win",
   "endGameLose": "end game as a loss",
   "endGameTooltip": "End the game as either a win or a loss.",
+  "extraFunction": "You have created a function called {funcName}, which is not needed for this puzzle. Check to see if you have a typo.",
```

Comment:
If I were optimising for the classroom scenarios we saw in those levels, I'd actually say "You have renamed a function to {funcName} but it won't work.  You should use the original function name."  

Totally get that the error listed is better for users who might add their own JS functions, but I think they'll be incredibly rare in levels 7-14.

@alicesteinglass might have thoughts.